### PR TITLE
doc(media): clarify the use of default media handlers

### PR DESCRIPTION
**Pull request by kgriffs**

### CODE REVIEW

File: docs/api/media.rst [hunk 1]
```diff
@@ -157,12 +157,20 @@ External handlers should update their logic to align to the internal Falcon hand
...
 receives `MessagePack <https://msgpack.org/>`_. We can easily do this by telling
-our Falcon API that we want a default media type of ``application/msgpack`` and
-then create a new :any:`Handlers` object specifying the desired media type and a
-handler that can process that data.
+our Falcon API that we want a default media type of ``application/msgpack``, and
+then creating a new :any:`Handlers` object to map that media type to an
+appropriate handler.
+
+The following example demonstrates how to replace the default handlers with an
+instance of Falcon's built-in MessagePack handler. However, you can always
```
Comment (vytas7) [inline docs/api/media.rst:172]: For the reader's convenience, we could link to our [`MessagePackHandler` class](https://falcon.readthedocs.io/en/latest/api/media.html#falcon.media.MessagePackHandler) here.

**Review by vytas7 — CHANGES_REQUESTED** [review]
LGTM with the exception of multipart form handler missing from the proposed sentence on the framework's default media handlers :+1:

File: docs/api/media.rst [hunk 2]
```diff
@@ -157,12 +157,20 @@ External handlers should update their logic to align to the internal Falcon hand
 Replacing the Default Handlers
 ------------------------------
 
+By default, the framework installs :class:`falcon.media.JSONHandler` and
+:class:`falcon.media.URLEncodedFormHandler` for the ``application/json``
+and ``application/x-www-form-urlencoded`` media types, respectively.
```
[the remark this answers is not in the document]
Comment (kgriffs) [reply docs/api/media.rst:162]: Yes, indeed.

File: docs/api/media.rst (hunk 1 above)
Comment (kgriffs) [reply docs/api/media.rst]: done

File: docs/api/media.rst (hunk 2 above)
[the remark this answers is not in the document]
Comment (kgriffs) [reply docs/api/media.rst]: done

**Review by vytas7 — APPROVED** [review]
:+1:

**Pull request merged**